.github/pull_request_template.md

```diff
@@ -2,34 +2,31 @@
   This template is not mandatory.
   It simply serves to provide a guide to allow a better review of pull requests.
 -->
-### What's is in this PR ?
+
+## What is in this PR?
```

Comment:
I'm sorry, but now that I've given this some thoughts, I am against that section too: we already have that information with the `changelog:*` labels. 
So not only does this section duplicates that information, but our labels have been set using standard changelog categories, and are used by our release tool to generate a standardized version's changelog automatically.